MetadataExtractor/Formats/Exif/ExifDescriptorBase.cs

```diff
@@ -271,6 +273,25 @@ public string GetResolutionDescription()
                 "cm");
         }
 
+        [CanBeNull]
+        public string GetPageNumberDescription()
+        {
+            var values = Directory.GetInt32Array(ExifDirectoryBase.TagPageNumber);
```

Comment:
Agree. We don't have a `GetInt16Array` extension method for `Directory` and I wasn't feeling up to making one, partly because of the inconsistent mess that permitting type coercion within directories has left us with. We could call `GetObject` and cast, but `GetInt32Array` will convert the value if it's non-standard, which does tend to increase coverage.